7zip.sls

```diff
@@ -31,7 +31,7 @@
 # An uninstall only definition will remain here so the packages will show up
 # correctly in `pkg.list_pkgs` and to allow for removal using `pkg.remove`
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-{% set versions = [ '18':['06', '05', '03', '01'], '16':['04', '03', '02', '00']} %}
+{% set versions = { '18':['06', '05', '03', '01'], '16':['04', '03', '02', '00']} %}
   {% for major, subversions in versions.items() %}
   {% for minor in subversions %}
     '{{major}}.{{minor}}.00.0':
```

Comment:
Probably too many indents here? Should only be 2 spaces. This might be causing this section not to be run by the tests... it would look like it was part of the last loop of the working definitions.